src/EditorFeatures/Core/Implementation/ReferenceHighlighting/AbstractDocumentHighlightsService.cs

```diff
@@ -130,8 +130,10 @@ private bool ShouldConsiderSymbol(ISymbol symbol)
             bool addAllDefinitions = true;
 
             // Add definitions
+            // Filter out definitions that cannot be highlighted. e.g: alias symbols defined via project property pages.
             if (symbol.Kind == SymbolKind.Alias &&
-                symbol.Locations.Length > 0)
+                symbol.Locations.Length > 0 &&
+                symbol.Locations.First().SourceTree != null)
```

Comment:
If the alias is defined in the project property pages, do we still need to set addAllDefinitions to false below?